### PR TITLE
authorize for job and company

### PR DESCRIPTION
**Pull request by quanglx-noovado**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## WHAT this PR does?
Phân quyền cho các controller

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by quanglx-noovado** [comment]
ready

File: app/Http/Controllers/CompanyController.php [hunk 1]
```diff
@@ -60,9 +60,9 @@ public function show($id)
      */
     public function edit($id)
     {
-        $company = Auth::user()->company;
-        
-        return $company->id == $id ? view('edit_company', compact('company')) : redirect()->route('home');
+        $company = Company::find($id);
+
```
Comment (congnx-2423) [inline app/Http/Controllers/CompanyController.php]: nếu không tìm thấy thì sao e nhỉ

**Review by huyenttm-0901 — COMMENTED** [review]
conflicted

**Comment by quanglx-noovado** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
conflict

**Comment by quanglx-noovado** [comment]
ready

**Pull request merged**